src/utils/ruleCalls.js

```diff
@@ -8,6 +8,14 @@ export const listRules = async () => {
   throw new Error("Error retrieving all rules");
 };
 
+export const mockListRules = async () => {
```

Comment:
Ahh for when we say we want mocked endpoints, it should match the path of the actual backend so you can still use the listRules function but the only thing that's changed is the base URL from your process env API_URL say to hit the mocked endpoint vs. the actual endpoint. we should still be using the listRules but only the baseURL would change depending on environment and the path like /rules should be the same as /bounce_rules and match the schema of the actual backend response